lib/editing/editor_comid.dart

```diff
@@ -7,33 +7,35 @@ library editor.comid;
 import 'dart:async';
 import 'dart:html' as html;
 import 'dart:math';
+import 'dart:convert';
 
 import 'package:comid/addon/comment/comment.dart' as comments;
 import 'package:comid/addon/edit/closebrackets.dart';
 import 'package:comid/addon/edit/matchbrackets.dart';
-import 'package:comid/addon/edit/show-hint.dart' as hints;
+import 'package:comid/addon/edit/show_hint.dart' as hints;
+import 'package:comid/addon/edit/html_hint.dart' as html;
 import 'package:comid/addon/mode/css.dart';
 import 'package:comid/addon/mode/dart.dart';
 import 'package:comid/addon/mode/htmlmixed.dart';
 import 'package:comid/codemirror.dart' hide Document;
+import "package:comid/addon/search/search.dart";
 
 import 'editor.dart' hide Position;
 import 'editor.dart' as ed show Position;
 
 export 'editor.dart';
 
-final ComidFactory comidFactory = new ComidFactory._();
+final CodeMirrorFactory comidFactory = new CodeMirrorFactory._();
```

Comment:
Using the name `ComidFactory` avoids confusion with the JS codemirror factory (also called `CodeMirrorFactory`).